packages/interface/src/features/home/components/Glossary.tsx

```diff
@@ -0,0 +1,71 @@
+import { Heading } from "~/components/ui/Heading";
+
+import { FAQItem } from "./FaqItem";
+
+export const Glossary = (): JSX.Element => (
```

Comment:
Actually am thinking about that, since most of the description is just a single sentence, should it be more like 3x3 cards instead of long page dropdown that require the user to click and open it? But this is just the styling thing, not affecting the functioning, maybe we could ask Mari for the answer.

Reply:
maybe we can have it open by design, but agree that maybe Mari can help us presenting it differently on a different place

Reply:
![image](https://github.com/user-attachments/assets/1c18efdd-4ad3-4464-8b98-aa551083be93)